### DISCUSSION TITLE
Day of defeat viewmodels for NZP [FTE/PC] (Unfinished) (MD3 format) (No PAP Variant)

### DISCUSSION THREAD

**Discussion by n4hw3ll** [nzp-team/nzportable#728] [Other Content Releases]
[nzp_md3viewmodels.zip](https://github.com/nzp-team/nzportable/files/15344306/nzp_md3viewmodels.zip)
here are some viewmodels i ported from Day of defeat to NZP for the players to enjoy, the pack's not finished as in there's missing weapons, some animations are not done and there's no PAP version of the gun

To install just move the "models" and "textures" folders inside the "nzp" folder

**Comment by totallypez** [comment]
been testing these in my install of nzp so far and they work pretty well !!! some things i noticed though are (sorry if this is a nitpick !!) the ironsights on the 1911 being slightly off but overall really good !! sprinting animations seem to replay part of the reload though for some reason,, other than that, i haven't had many problems with these !! imo you should definitely try to get more of the dod models into nzp, they overall really fit the gameplay and style, and look very nice to play with !!! i need to figure this type of stuff myself though seriously !!! it's super inspiring !!! :oo
